shared/hybrid/AppDelegate+SalesforceHybridSDK.m

```diff
@@ -111,15 +125,17 @@ - (void)sfsdk_swizzled_application:(UIApplication *)application didRegisterForRe
     [self sfsdk_swizzled_application:application didRegisterForRemoteNotificationsWithDeviceToken:deviceToken];
 }
 
-- (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options{
-
-    // If you're using advanced authentication:
+- (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options
+{
+    
+    //Uncomment following block to enable IDP Login flow or If you're using advanced authentication:
     // --Configure your app to handle incoming requests to your
     //   OAuth Redirect URI custom URL scheme.
     // --Uncomment the following line and delete the original return statement:
-
-    // return [[SFUserAcountManager sharedInstance] handleAdvancedAuthenticationResponse:url options:options];
-    return NO;
+    /*
+     return [[SFUserAccountManager sharedInstance] handleAdvancedAuthenticationResponse:url options:options];
+     // */ return NO;
```

Comment:
The commenting out here is kinda weird. Can we either remove the extra `//` or `/* */`?